ams/server/src/main/java/com/netease/arctic/server/dashboard/DashboardServer.java

```diff
@@ -186,6 +186,9 @@ public EndpointGroup endpoints() {
             get(
                 "/tables/catalogs/{catalog}/dbs/{db}/tables/{table}/optimizing-processes",
                 tableController::getOptimizingProcesses);
+            get(
+                "/tables/catalogs/{catalog}/dbs/{db}/tables/{table}/optimizing-processes/{processId}/detail",
```

Comment:
Maybe the Http API can be changed to `/tables/catalogs/{catalog}/dbs/{db}/tables/{table}/optimizing-processes/{processId}/tasks`, because this API is supposed to return a list of optimizing task informations.

Reply:
Done.